apps/examples/src/examples/changing-default-colors/README.md

```diff
@@ -3,6 +3,7 @@ title: Changing default colors
 component: ./ChangingDefaultColorsExample.tsx
 category: ui
 priority: 1
+keywords: [colours, styles, palette, style panel]
```

Comment:
heh, we prbly want "color", at least in addition :)

Reply:
I think that's covered by the title, it's in the keywords as a catch-all